pkg/prometheus_metrics/metrics.go

```diff
@@ -0,0 +1,28 @@
+package metrics
+
+import (
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// MetricName for stork prometheus metrics
+	MetricName = "name"
+	// MetricNamespace for stork prometheus metrics
+	MetricNamespace = "namespace"
+	// MetricSchedule for stork prometheus metrics
+	MetricSchedule = "schedule"
+)
+
+func init() {
+	if err := storkops.Instance().WatchApplicationBackup("", watchBackupCR, metav1.ListOptions{}); err != nil {
```

Comment:
Move these to the init in the specific file

Reply:
Since we want to start/stop metrics using params added `StartMetrics` watcher as func  and called it via cmd/stork.go to enable/disable metrics support